dask/dataframe/io.py

```diff
@@ -462,14 +462,18 @@ def to_hdf(df, path_or_buf, key, mode='a', append=False, complevel=0,
                  "must preserve the order of its input")
 
     # handle lock default based on wether we're writing to a single entity
+    _actual_get = get if get else dask.get
```

Comment:
`_globals` doesn't always hold a get, so I recommend something like the following:

    get = get or _globals.get('get') or get_sync